src/main/java/uk/gov/pay/api/model/PaymentProvider.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.pay.api.model;
+
+public enum PaymentProvider {
```

Comment:
Do we need this? Won't we just pass on the string returned by dd-connector. Or is this considered better?

Reply:
oh at the moment dd-connector doesn't return the payment provider name. i'll need to amend that